libcxx/include/fstream

```diff
@@ -227,6 +227,8 @@ _LIBCPP_EXPORTED_FROM_ABI void* __filebuf_windows_native_handle(FILE* __file) no
 
 template <class _CharT, class _Traits>
 class basic_filebuf : public basic_streambuf<_CharT, _Traits> {
+  using __base _LIBCPP_NODEBUG = basic_streambuf<_CharT, _Traits>;
```

Comment:
Let's use `basic_streambuf` directly and avoid introducing this typedef which is used only once.